onnxruntime/contrib_ops/cpu/murmur_hash3.cc

```diff
@@ -193,14 +197,14 @@ Status MurmurHash3::Compute(OpKernelContext* ctx) const {
       ++output;
     }
   } else {
-    auto input = reinterpret_cast<const uint32_t*>(keys->DataRaw());
-    const auto input_end = input + input_count;
+    auto input = reinterpret_cast<const unsigned char*>(keys->DataRaw());
+    const auto input_end = input + input_count * static_cast<int>(input_element_bytes);
     while (input != input_end) {
       MurmurHash3_x86_32(input,
                          static_cast<int>(input_element_bytes),
                          seed_,
                          output);
-      ++input;
+      input += static_cast<int>(input_element_bytes);
```

Comment:
Convert once `static_cast<int>(input_element_bytes)` in the beginning and reuse its value.